frontend/src/Components/BusinessDetails/BusinessDetails.jsx

```diff
@@ -67,13 +77,33 @@ const BusinessDetails = () => {
     setReviews((prevReviews) => [...prevReviews, newReview]);
     setShowReviewForm(false);
   };
+  const parseLocation = (location) => {
+    const [lat, lng] = location
+      .split(",")
+      .map((coord) => parseFloat(coord.trim()));
+    return { lat, lng };
+  };
+
+  const handleClick = (event) => {};
+
+  const handleMouseOver = () => {
```

Comment:
it seems like we only open the info window when mousing over a marker on the map. However, this might be inaccessible to users with a physical impairment who don't use a mouse.  Should we also open the info window on click?

Reply:
I'll consider this